### PR TITLE
Added S3 bucket for email enrolment images

### PR DESCRIPTION
**Pull request by webbyfox**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: conf/settings.py [hunk 1]
```diff
@@ -312,6 +312,11 @@
 AWS_STORAGE_BUCKET_NAME_DATA_SCIENCE = env.str('AWS_STORAGE_BUCKET_NAME_DATA_SCIENCE', '')
 AWS_S3_REGION_NAME_DATA_SCIENCE = env.str('AWS_S3_REGION_NAME_DATA_SCIENCE', '')
 
+# Setting for email buckets which holds images
+AWS_ACCESS_KEY_ID_EMAIL = env.str('AWS_ACCESS_KEY_ID_EMAIL', '')
+AWS_SECRET_ACCESS_KEY_EMAIL = env.str('AWS_SECRET_ACCESS_KEY_EMAIL', '')
+AWS_STORAGE_BUCKET_NAME_EMAIL = env.str('AWS_STORAGE_BUCKET_NAME_EMAIL', '')
```
Comment (depsiatwal) [inline conf/settings.py:318]: Why do we need another bucker, we now have 3 in total :-) 
I thinking we should make do with 1

**Review by depsiatwal — COMMENTED** [review]
see inline comment

File: conf/settings.py (hunk 1 above)
Comment (webbyfox) [reply conf/settings.py:318]: Good point, Don't think we can merge into one - as we have data science bucket. we already had three buckets - as directory-email-static bucket was hardcoded in constant.py, so using this opportunity to move in to vault. This change we are doing to move that bucket from Ireland(eu-west-1) to London (eu-west-2) but let me check with Jayesh if they have same permission level then we can move those email images to existing bucket

Comment (webbyfox) [reply conf/settings.py:318]: checked with Jayesh - That EMAIL bucket is only one bucket for all environments and those existing buckets have separate bucket for each environment

**Review by depsiatwal — APPROVED** [review]
As discussed all Good 
bucket is using an already existing bucket this is just moving it to config

**Pull request merged**